### ISSUE TITLE
[JAWS 18] The associated titles/captions of forms and regions are not announced

### ISSUE THREAD

**Issue by NikolayCholakov** [FreedomScientific/VFO-standards-support#34]
### Summary

> JAWS doesn't announce the associated titles/captions of forms and regions when navigating to one of its interactive elements

### Example

> 1. Go to https://codepen.io/anon/pen/xpxqod
> 2. There are two examples:
>      - simple form
>      - nested forms
> 3. Try navigating one of their interactive elements using TAB key

### Expected result

> Entering a form region with its associated name to be announced before reading the information of the currently focused interactive element.

### Actual result

> 1. Simple form - only information about the focused element is announced by JAWS. Form information and its name are omitted.
> 2. Nested forms - only information about the focused element is announced by JAWS. Main and nested form information and their names are omitted.

### Additional Information

This example was tested with the following screen reader and browser combinations:

> 1. JAWS + Chrome - works
> 2. JAWS + IE11 - doesn't work
> 3. NVDA + Chrome - works
> 4. NVDA + IE11 - works

#### JAWS version and build number
 
JAWS 18.0.4352.400

#### Operating System and version

Windows 10 Enterprise version 1607 (build 14393.1944)

#### Browser and version:

Internet Explorer 11.1944.14393.0 (update versions 11.0.49)

**Comment by stevefaulkner** [comment]
Hi @NikolayCholakov the markup pattern you are using 
`<fieldset role="form">` is [non-conforming](https://w3c.github.io/html-aria/#fieldset). the only allowed meaningful role is `group`. If you use `role="group"` it works as expected in IE. [Test case](https://codepen.io/stevef/pen/jYLwZP)
